backend/modules/load/services/queries/ports/repositories.py

```diff
@@ -0,0 +1,14 @@
+from abc import abstractmethod
+from typing import List
+
+from .....common.domain import ports as common_ports
+from .....data.domain import value_objects as data_value_objects
+
+
+class AbstractDataQueryRepository(common_ports.AbstractDomainRepository):
+    @abstractmethod
+    def list(self) -> tuple[List[data_value_objects.OutputData], int]:
```

Comment:
Missing returns type hint

Reply:
The return type is tuple[List[data_value_objects.OutputData], int]

Am I missing something obvious ?

Reply:
No no, my bad. Mobile Github nested it wrongly and I did not notice that.